deploy/frontend.yaml

```diff
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/RedHatInsights/frontend-components/refs/heads/master/packages/config-utils/src/feo/spec/frontend-crd.schema.json
+---
 apiVersion: v1
 kind: Template
```

Comment:
do we not need to add the feoEnabled param?
```
  - spec:
      feoConfigEnabled: true
```

Reply:
@Hyperkid123 ^

Reply:
@aferd You're right! I forgot to push the change